Makefile

```diff
@@ -12,6 +12,8 @@ LOOP_PRIVACY_WEBSITE_URL := $(shell echo $${LOOP_PRIVACY_WEBSITE_URL-"https://ww
 LOOP_LEGAL_WEBSITE_URL := $(shell echo $${LOOP_LEGAL_WEBSITE_URL-"https://www.mozilla.org/about/legal/terms/firefox-hello/"})
 LOOP_PRODUCT_HOMEPAGE_URL := $(shell echo $${LOOP_PRODUCT_HOMEPAGE_URL-"https://www.firefox.com/hello/"})
 FIREFOX_VERSION=48.0
+# Taken from http://hg.mozilla.org/mozilla-central/file/5e14887312d4//old-configure.in#l8032
+MOZ_APP_MAXVERSION=`echo $(FIREFOX_VERSION) | sed "s|\(^[0-9]*\).*|\1|"`.*
```

Comment:
Are we planning on using `FIREFOX_VERSION` for something? It's passed in to the preprocessor as well but not used in `install.rdf.in`. So perhaps just have a `FIREFOX_MAXVERSION=48.*` to not need the `sed`/comment.